src/helpers/convert-direction.js

```diff
@@ -0,0 +1,13 @@
+import Swing from 'react-swing';
+
+    const voteTypeOf = dir => {
+      const voteType= {};
+      voteType[Swing.DIRECTION.DOWN]= false;
```

Comment:
false and null are both falsy values, is it worth us using 2 truthy values to show how people have voted?

Reply:
+1 and -1?

Reply:
Sounds good 👍 👎 👍